### DISCUSSION TITLE
Should width be the minimum of `-w` parameter and terminal width?

### DISCUSSION THREAD

**Discussion by J-Kappes** [eza-community/eza#831] [Opinion Polls]
Personally, I have `ls` aliased to `echo;eza --icons --no-quotes -w 110`. I use the `-w` for readability by limiting the width on wide terminals.
However, using that alias in a small terminal can result in wrapping. I would prefer it if the `-w` parameter only presented an upper bound to the width instead of overriding the terminal width. What about you?

**Comment by daviessm** [comment]
I'd be happy with a `--max-width` parameter.

**Reply by cafkafk** [reply]
I also prefer this tbh, that way we don't break backwards compatibility (although you could argue the previous -w behavior isn't super useful)
